examples/text_classification-tf.ipynb

```diff
@@ -84,7 +84,7 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    pipe = pipeline("text-classification", "Rocketknight1/distilbert-base-uncased-finetuned-cola", framework="tf")
</pre><p>Let's call this <code>classifier</code> or something along those lines. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>